Firmware/RTK_Surveyor/Begin.ino

```diff
@@ -59,8 +132,8 @@ void beginBoard()
     //The following changes the POR default but will be overwritten by settings in NVM or settings file
     settings.ubxConstellations[1].enabled = false;
 
-    strcpy(platformFilePrefix, "SFE_Surveyor");
-    strcpy(platformPrefix, "Surveyor");
+    strncpy(platformFilePrefix, "SFE_Surveyor", sizeof(platformFilePrefix) - 1);
```

Comment:
Thank you! Please help us move completely to strncpy.

Reply:
Happy to! :-)